src/states/features/product/productSlice.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-use-before-define */
+
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import productServices from './productService';
+
+const currentToken = localStorage.getItem('myToken');
+
+const initialState = {
+  token: currentToken,
+  isError: false,
+  isSuccess: false,
+  isLoading: false,
+  message: ''
+};
```

Comment:
This is wrong, this is the product initial state how are you sitll bringing in tokens? there should be things related to products